HECalculator/Backend/Worker.cs

```diff
@@ -5,4 +5,6 @@ public class Worker
 	public Worker()
 	{
 	}
+
+	private Inputs inputs;
```

Comment:
Add generic getter and setter

Reply:
Should discuss a standard of making getters and setters (auto property, traditional...)

Reply:
Discussion can be found here #13